frontend/src/console/components/PreviewComponent/styles.scss

```diff
@@ -1,4 +1,7 @@
 .theme-preview {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
Can we use Bootstrap CSS classes instead? [Documentation](https://getbootstrap.com/docs/4.0/utilities/flex/).

Reply:
Thanks for this tip @0x29a . This helps me in other tasks as well.